db/schema.rb

```diff
@@ -1778,6 +1783,7 @@
   add_foreign_key "greeter_organization_join_records", "greeter_roles"
   add_foreign_key "greeter_organization_join_records", "vita_partners"
   add_foreign_key "incoming_text_messages", "clients"
+  add_foreign_key "intake_archives", "intakes", column: "id"
```

Comment:
Seems to add confusion/complexity to make the intake_archive use the intake id but maybe its fancy and we'd like it